util/chunk/chunk.go

```diff
@@ -174,6 +186,7 @@ func (c *Chunk) Append(other *Chunk, begin, end int) {
 			dst.length++
 		}
 	}
+	c.numVirtualRows += other.numVirtualRows
```

Comment:
why add all other.numVirtualRows? if we call Append more than once

Reply:
a mistake, fixed